zmdocument.cls

```diff
@@ -247,8 +239,8 @@
    \setcounter{@authorsNumber}{\the@listSize}
    \setcounter{@columnNumber}{\the\numexpr(1 + \the@authorsNumber / \the@authorsByColumn)\relax}
    \thispagestyle{empty}
-   \pagecolor{bgtitle}
-   \color{texttitle}
+   \pagecolor{titlePageBackgroundColor}
+   \color{titlePageTextColor}
```

Comment:
Set these three last lines in \maketitle macros (they don’t concern authors formatting).